### PR TITLE
fix: no output issue

### LINKED ISSUE #731: pulumi output are not display during pulumi up
### What happened?

During the pulumi up action, if there is no resources to modify there is nothing display

### Steps to reproduce

1. Launch `pulumi up` when no resources are require modifications.

### Expected Behavior

pulumi_up-3 18 1

### Actual Behavior

pulumi_up-3 19 1

### Output of `pulumi about`

_No response_

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### PR DESCRIPTION
**Pull request by cobraz**
No clue why tests are failing 🤷‍♂️

might fix #731

### CODE REVIEW

**Comment by cobraz** [comment]
@RobbieMcKinstry Would you mind having a look at this? Do you see why the tests are failing?

**Comment by RobbieMcKinstry** [comment]
@cobraz Sure, let me take a look...

**Comment by RobbieMcKinstry** [comment]
@cobraz Just wondering, are you not able to see the Actions logs? For example, does [this link](https://github.com/pulumi/actions/actions/runs/3199973640/jobs/5226350091#step:6:18) work for you? That's the link to the test failure in CI.

Screen Shot 2022-10-07 at 1 39 42 PM

I'm seeing type error in the logs: The test failure claims `getExecOutput` isn't a function.

**Comment by cobraz** [comment]
Oh, yes. I've seen this issue, but I don't know why. I don't wanna bug you with this, just wanted a second pair of eyes

**Comment by RobbieMcKinstry** [comment]
Understood! Don't worry about bugging me, it's no bother at all :) I just wanted to make sure I understood the ask. Happy to lend a second set of eyes! I'm going to pull down the code locally now. 👍🏻  I appreciate you taking a stab at #731.

**Comment by RobbieMcKinstry** [comment]
I'm sorry if my original comment came off as rude. I didn't mean to sound dismissive or curt! Apologies!

**Comment by RobbieMcKinstry** [comment]
Just a quick update. I pushed a change that fixed the broken test.

`exec` was mocked out in Jest, so switching to `getExecOutput` required the mock to be updated.

**Comment by cobraz** [comment]
Let's land this anyway?

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "@actions/core": "^1.10.0",
-    "@actions/exec": "^1.1.0",
+    "@actions/exec": "^1.1.1",
```
Comment (cobraz) [inline package.json:46]: This is probably not needed, but hey, we like bumping dependencies don't we!

**Comment by RobbieMcKinstry** [comment]
Holy moly, I'm sorry I missed the notification for this! Very sorry about that!

**Review by RobbieMcKinstry — APPROVED** [review]

**Pull request merged**